src/impl/MilvusClientImpl.cpp

```diff
@@ -462,7 +463,39 @@ MilvusClientImpl::Search(const SearchArguments& arguments, const SearchResults&
 }
 
 Status
-MilvusClientImpl::Query(const QueryArguments& arguments, const QueryResults& results) {
+MilvusClientImpl::Query(const QueryArguments& arguments, QueryResults& results) {
+    if (connection_ == nullptr) {
+        return {StatusCode::NOT_CONNECTED, "Connection is not ready!"};
+    }
+    proto::milvus::QueryRequest rpc_request;
+    rpc_request.set_collection_name(arguments.CollectionName());
+    const auto& partition_names = arguments.PartitionNames();
+    auto& names = *(rpc_request.mutable_partition_names());
+    for (auto partition_name : partition_names) {
+        names.Add(std::move(partition_name));
+    }
+
+    rpc_request.set_expr(arguments.Expression());
+    const auto& output_fields = arguments.OutputFields();
+    auto& fields = *(rpc_request.mutable_output_fields());
+    for (auto field : output_fields) {
+        fields.Add(std::move(field));
+    }
+
+    rpc_request.set_travel_timestamp(arguments.GuaranteeTimestamp());
```

Comment:
Here should be "TravelTimestamp()"

Reply:
Fixed with add argument matcher in testing code.